Projects/Embark/Sources/Viewables/Actions/Masking.swift

```diff
@@ -14,8 +14,25 @@ enum MaskType: String {
 struct Masking {
     let type: MaskType
 
-    func isValid(text _: String) -> String {
-        ""
+    func isValid(text: String) -> Bool {
+        switch type {
+        case .personalNumber:
+            return text.count > 10
+        case .birthDate, .birthDateReverse:
+            return text.count == 10
```

Comment:
if the formatting is slightly different from be won't this fail? I saw on the Moving Flow - birthdate was formatted without dashes. Don't we get the max min from BE for validation?

Reply:
The function here makes sure the formatting is always correct: https://github.com/HedvigInsurance/ugglan/blob/e967efe69cb87d508897852681ae6172ed28c75a/Projects/Embark/Sources/Viewables/Actions/Masking.swift#L176

I am guessing in the case of moving flow a mask hasn't been used/set at all in angel, but in that case this function wouldn't be called at all 😄 We should also implement min/max though.